content.js

```diff
@@ -36,9 +36,15 @@ if (matchPattern.test(url)) {
 function run() {
     let videoDiv = document.getElementById('video_0');
     let videoUrl = videoDiv.firstElementChild.getAttribute('src');
+    let fileName = new URL(videoUrl).pathname.split("/").slice(-1)[0];
+    let fileExtension = "." + fileName.split(".").slice(-1)[0];
+    let title = document.title;
+    if (!title.endsWith(fileExtension)) {
```

Comment:
I found cases where there was a mismatch between the file extension in the url and the title.
For example in the course Datenbankeinsatz WS 20/21 the first video has the title ```2020-11-04 07-58-34 Edit-1a.m4v``` while the filename in the link is ```o_1em9aadjfemirq05brc911tls7.mp4```.
I think it would be better to check if the file ends with any file extension, not only the one from the url.
The question remaining is whether we should use the extension from the title or the filename in case of a mismatch.

Reply:
Huh that's..interesting. I guess most video formats are somewhat interchangeable anyway, so it probably does not matter too much which one we use. In the example you mentioned both extensions can be used without any problems.
I would think that the filename in the video url would have a higher chance of being the correct one, as this is probably directly related to the original file and probably also used by the browser when playing back the video. But I have no idea if that is actually the case.
On the other side, I don't think it would be too bad to even just keep the current behavior, where the file extension from the video url is always appended. In your example, `2020-11-04 07-58-34 Edit-1a.m4v.mp4` would still be playable without any problems. Further, if the wrong extension was indeed appended, then the user could still change it back to the other one.

Let me know which option you prefer and I'll perform the necessary changes. :)

Reply:
You make a good point with your last paragraph. While I was first skeptical, I can see the benefit of providing this extra information instead of removing it all together and potentially causing issues for the end user.
I guess we'll leave it at this implementation then.

Thanks again for the PR!